src/storage/oos_file.hpp

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2008 Search Solution Corporation
+ * Copyright 2016 CUBRID Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+#pragma once
+
+#include "dbtype_def.h"
+#include "storage_common.h"
+#include "thread_compat.hpp"
+
+struct oos_record_header
+{
+  int total_size;
+  int chunk_index;
+  OID next_chunk_oid;
+};
+using OOS_RECORD_HEADER = struct oos_record_header;
```

Comment:
아래 코드 블럭은 외부에서 참조되지 않습니다. oss_file.cpp로 옮겨도 될 것 같습니다.
```suggestion
struct oos_record_header
{
  int total_size;
  int chunk_index;
  OID next_chunk_oid;
};
using OOS_RECORD_HEADER = struct oos_record_header;
```

Reply:
처리했습니다.
<img width="1651" height="946" alt="image" src="https://github.com/user-attachments/assets/9c366b83-d4c5-4352-9ebf-4d4e07e0645b" />